docs/runbooks/HCOInstallationIncomplete.md

```diff
@@ -13,19 +13,21 @@ uninstalling the HCO and the HCO is still running.
 
 ## Mitigation
 
-Installation: Complete the installation by creating a `HyperConverged` CR with
-its default values:
+The mitigation depends on whether you are installing or uninstalling
+the HCO:
 
-```bash
-$ cat <<EOF | kubectl apply -f -
-apiVersion: operators.coreos.com/v1
-kind: OperatorGroup
-metadata:
-   name: hco-operatorgroup
-   namespace: kubevirt-hyperconverged
-spec: {}
-EOF
-```
+- Complete the installation by creating a `HyperConverged` CR with its
```

Comment:
This matches what we have downstream, but I feel it could use more clarification.

```suggestion
- Complete the HCO installation by creating a `hco-operatorgroup` CR with
```

Reply:
^ and this might also need to be fixed downstream, if I am correct

Reply:
the idea of this PR is to close the gap between downstream and upstream to enable the sync automation

Reply:
and I'm not even sure about this step, and it's confusing, HyperConverged CR, OperatorGroup...

/cc @nunnatsa @tiraboschi

Reply:
Currently (OLM v0) installing an operator managed application requires basically 3 main distinct steps:
1. Creating the operatorGroup
2. Creating the subscription object to have the OLM deploying the operator.
Steps 1. and 2. are documented in https://docs.openshift.com/container-platform/4.15/virt/install/installing-virt.html#virt-subscribing-cli_installing-virt
3. Creating a CR object (the hyperconverged CR in this case) for the operator to let it complete the deployment of the application (OpenShift Virtualization in this case). This step is documented in https://docs.openshift.com/container-platform/4.15/virt/install/installing-virt.html#virt-deploying-operator-cli_installing-virt

This specific alert (`HCOInstallationIncomplete`) indicates that steps 1 and 2 got correctly completed (the operator is there and so its able to fire the alert) but it's still waiting for an HyperConverged CR object to complete the deployment.
So technically `- Complete the installation by creating a HyperConverged CR with its...` was accurate but the suggested content should be:
```
apiVersion: hco.kubevirt.io/v1beta1
kind: HyperConverged
metadata:
  name: kubevirt-hyperconverged
  namespace: openshift-cnv
spec: {}
```

The OperatorGroup and Subscription should be first deleted and recreated only if the cluster admin wants to restart from scratch but this is not strictly needed nor recommended.

Reply:
I suspected, updated. @ousleyp please take another look, I think with the HCO CR this description now makes sense

Reply:
@tiraboschi @machadovilaca Is the namespace `openshift-cnv` correct to use in the kubevirt/monitoring runbooks?

Otherwise, that makes a lot more sense; thanks for clarifying! I saw that this is also incorrect in [the downstream docs](https://docs.openshift.com/container-platform/4.15/virt/monitoring/virt-runbooks.html#virt-runbook-HCOInstallationIncomplete)... I'll open a PR to fix it in the doc versions where we're keeping the runbooks published.

Reply:
since we will be syncing this to downstream runbooks, maybe we can just use a filler like: `<namespace>`

Reply:
Thanks for reviewing the openshift/monitoring PR! 

I also opened the following PR to fix this in the downstream docs: https://github.com/openshift/openshift-docs/pull/75888